game.js

```diff
@@ -17,7 +17,7 @@ var fallenwoodCounter = function() {
 
 var setEventHandlers = function() {
 	$('#click-here-button').click(PickUpWoodButtonClick);
-	setInterval(secondCounterHandler, 1000);
+	setInterval(one second timer, 1000);
```

Comment:
Variable/function names can't have spaces. There are also two places this needs to be changed